torch/csrc/autograd/profiler.cpp

```diff
@@ -39,7 +42,11 @@ void mark(std::string name, bool include_cuda /* = true */) {
   }
 }
 
-void pushRange(std::string name, const char* msg/*= ""*/, int64_t sequence_nr/*= -1*/) {
+const char* c_str(const char *str) { return str; }
+const char* c_str(const std::string& str) { return str.c_str(); }
```

Comment:
Hmmm I think you're right... I'll fix that.

Reply:
Actually it's ok in the way it's used, because its argument outlives the call. I'll still change it to be a non-const reference, to ensure that temporaries cannot bind to it.